.chloggen/ingest_tls_openshift.yaml

```diff
@@ -0,0 +1,28 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, github action)
+component: operator
```

Comment:
Could you please rebase and change this?

Reply:
Done